tools/cldr-apps/js/src/esm/cldrPriorityItems.mjs

```diff
@@ -137,14 +137,9 @@ function createSnapshot(summarizeAllLocales) {
 
 function requestSummary(summaryArgs) {
   latestArgs = summaryArgs;
+  const init = cldrAjax.makePostData(summaryArgs);
   cldrAjax
-    .doFetch(SUMMARY_URL, {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify(summaryArgs),
-    })
+    .doFetch(SUMMARY_URL, init)
```

Comment:
changes in this file are only for refactoring with makePostData, to encapsulate POST, headers, stringify